### PR TITLE
Lipid search database table fix

### PR DESCRIPTION
**Pull request by ansgarkorf**
FXing for Lipid Search module

### CODE REVIEW

File: src/main/java/io/github/mzmine/modules/dataprocessing/id_lipididentification/LipidSearchMSMSParameters.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package io.github.mzmine.modules.dataprocessing.id_lipididentification;
```
Comment (tomas-pluskal) [inline src/main/java/io/github/mzmine/modules/dataprocessing/id_lipididentification/LipidSearchMSMSParameters.java:19]: Please add the license header

File: src/main/java/io/github/mzmine/modules/dataprocessing/id_lipididentification/lipids/lipidmodifications/LipidModificationChoiceComponent.java [hunk 2]
```diff
@@ -163,7 +161,7 @@ public LipidModificationChoiceComponent(LipidModification[] choices) {
       checkList.getItems().clear();
     });
 
-
+    System.out.println(checkList.getSelectionModel().getSelectedItems().size());
```
Comment (tomas-pluskal) [inline src/main/java/io/github/mzmine/modules/dataprocessing/id_lipididentification/lipids/lipidmodifications/LipidModificationChoiceComponent.java]: Remove println

File: src/main/java/io/github/mzmine/modules/dataprocessing/id_lipididentification/lipids/TableModel.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
+package io.github.mzmine.modules.dataprocessing.id_lipididentification.lipids;
```
Comment (tomas-pluskal) [inline src/main/java/io/github/mzmine/modules/dataprocessing/id_lipididentification/lipids/TableModel.java]: Please add the license header

File: src/main/java/io/github/mzmine/modules/dataprocessing/id_lipididentification/lipids/TableModel.java [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+  private String molecularFormula;
+  private String abbreviation;
+  private String ionization;
+  private String exactMass;
+  private String info;
+  private String status;
+  private String msmsFragmentsPos;
+  private String msmsFragmentsNeg;
+
+  public TableModel(String id, String lipidCoreClass, String lipidMainClass, String lipidClass,
```
Comment (tomas-pluskal) [inline src/main/java/io/github/mzmine/modules/dataprocessing/id_lipididentification/lipids/TableModel.java]: I think the "TableModel" name is too generic. Maybe this class could be named "LipidClassDescription" or something like that?

**Comment by ansgarkorf** [comment]
All done, thanks for revision and sorry for the license...

**Comment by tomas-pluskal** [comment]
Thanks Ansgar!

**Pull request merged**